potpie/cli.py

```
@@ -0,0 +1,313 @@
+    else:
+        try:
+            os.killpg(tracked_process.pid, signal.SIGTERM)
+            typer.echo(f"Stopped Potpie process group for PID {tracked_process.pid}.")
+        except ProcessLookupError:
+            typer.echo("No tracked Potpie process is running.")
+        except PermissionError:
+            typer.echo(f"Permission denied while stopping PID {tracked_process.pid}.")
+
+    PID_FILE.unlink(missing_ok=True)
+
+    subprocess.run(["make", "infra-down"], check=False)
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)